src/extensions/default/AutoUpdate/main.js

```diff
@@ -140,6 +153,27 @@ define(function (require, exports, module) {
         updateDomain.exec('data', msg);
     }
 
+     /**
+     * Checks Install failure scenarios
+     */
+    function checkInstallationStatus() {
+        var searchParams = {
+                "updateDir": updateDir,
+                "installErrorStr": ["ERROR:"],
+                "bracketErrorStr": ["ERROR:"],
+                "encoding": "utf8"
+            },
+            //Below are possible Win Installer error string which will be searched installer logs to track failure
```

Comment:
nit: `// Below are the possible Windows Installer error strings, which will be searched for in the installer logs to track failures`

Reply:
We are not tracking failures for Mac?

Reply:
we are tracking using the Search String "ERROR:". This string is default. so Only in windows case I explicitly set the search String